<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZoomCameraTestWidget</class>
 <widget class="QWidget" name="ZoomCameraTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZoomCameraTestWidget</string>
  </property>
  <widget class="QPushButton" name="pushButton_Connect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Disconnect">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Stop">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Connect</sender>
   <signal>released()</signal>
   <receiver>ZoomCameraTestWidget</receiver>
   <slot>onButtonConnectSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Disconnect</sender>
   <signal>released()</signal>
   <receiver>ZoomCameraTestWidget</receiver>
   <slot>onButtonDisconnectSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Start</sender>
   <signal>released()</signal>
   <receiver>ZoomCameraTestWidget</receiver>
   <slot>onButtonStartSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Stop</sender>
   <signal>released()</signal>
   <receiver>ZoomCameraTestWidget</receiver>
   <slot>onButtonStopSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onButtonConnectSlot()</slot>
  <slot>onButtonDisconnectSlot()</slot>
  <slot>onButtonStartSlot()</slot>
  <slot>onButtonStopSlot()</slot>
 </slots>
</ui>
